cartoframes/data/services/isolines.py

```diff
@@ -161,7 +161,7 @@ def _iso_areas(self,
             'maxpoints': maxpoints,
             'quality': quality
         }
-        iso_options = [str(k)+'='+str(v) for k, v in options.items() if v is not None]
+        iso_options = ["'"+str(k)+'='+str(v)+"'" for k, v in options.items() if v is not None]
```

Comment:
You can use this instead `"'{}={}'".format(k, v)`

Reply:
totally